src/components/course-card/components/PreviewVideo.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from '../styles.module.css';
+import { CourseVideoPreview } from '../../../types/course.type';
```

Comment:
It would be better to locate import that have more nested path above less nested, so it should looks like this:

import { CourseVideoPreview } from '../../../types/course.type';
import styles from '../styles.module.css';